server/src/utils.ts

```diff
@@ -128,7 +144,8 @@ export let formatUsingValidBscNativePath = (
 export let runAnalysisAfterSanityCheck = (
   filePath: p.DocumentUri,
   args: Array<any>,
-  projectRequired=false
+  projectRequired = false,
+  parseJson = true
```

Comment:
The `createInterface` command, for example, does the json escape in `Cli.ml`, so we don't need to special case the command here.